### PR TITLE
Get Mentions

### PR DESCRIPTION
**Pull request by aliyss**
I added it like this, but honestly it isn't really optimal. The first set returns id's only. The function converts the ids to Contacts and returns it as an array of Contacts. Do you see a more likeable solution?

### CODE REVIEW

**Comment by Aliyss** [comment]
On the other hand I could make it it's own object. As a collection of Contacts?

File: src/structures/Message.js [hunk 1]
```diff
@@ -90,6 +90,16 @@ class Message extends Base {
          */
         this.hasQuotedMsg = data.quotedMsg ? true : false;
 
+        /**
+         * Indicates the mentions in the message body.
+         * @type {[id]}
```
Comment (pedroslopez) [inline src/structures/Message.js]:
```suggestion
         * @type {Array<string>}
```

File: src/structures/Message.js [hunk 2]
```diff
@@ -113,6 +123,19 @@ class Message extends Base {
         return this.client.getContactById(this._getChatId());
     }
 
+    /**
+     * Returns the Contacts mentioned in this message
+     * @returns {[Promise<Contact>]}
```
Comment (pedroslopez) [inline src/structures/Message.js]:
```suggestion
     * @returns {Promise<Array<Contact>>}
```

File: src/structures/Message.js [hunk 3]
```diff
@@ -90,6 +90,16 @@ class Message extends Base {
          */
         this.hasQuotedMsg = data.quotedMsg ? true : false;
 
+        /**
+         * Indicates the mentions in the message body.
+         * @type {[id]}
+         */
+        this.mentions = [];
```
Comment (pedroslopez) [inline src/structures/Message.js]: Rename `mentions` to `mentionedIds` to indicate the difference between this and the other function.

**Review by pedroslopez — CHANGES_REQUESTED** [review]
This looks fine to me! Just a couple comments.

File: src/structures/Message.js [hunk 4]
```diff
@@ -113,6 +123,19 @@ class Message extends Base {
         return this.client.getContactById(this._getChatId());
     }
 
+    /**
+     * Returns the Contacts mentioned in this message
+     * @returns {[Promise<Contact>]}
+     */
+    async getMentions() {
+        let mentions = [];
+        for (let i = 0; i < this.mentions.length; i++) {
```
Comment (pedroslopez) [inline src/structures/Message.js]: I would prefer using `map` over loops like these. Makes it clearer.

**Comment by Aliyss** [comment]
Okok will push the changes in the morning.

**Comment by Aliyss** [comment]
Doesn't give an error if own user is mentioned, but doesn't return clean data.

**Pull request merged**